.buildkite/bk.integration.pipeline.yml

```diff
@@ -21,6 +21,74 @@ steps:
       image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-beats-ci-with-hooks:0.5"
       useCustomGlobalHooks: true
 
+  - label: Start Serverless project
+    key: integration-serverless
+    env:
+      ASDF_TERRAFORM_VERSION: 1.9.2
+    command: |
+      #!/usr/bin/env bash
+      set -euo pipefail
+      source .buildkite/scripts/steps/serverless_start.sh
+    artifact_paths:
+      - test_infra/serverless/*.tfstate
+      - test_infra/serverless/*.lock.hcl
+    agents:
+      image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-beats-ci-with-hooks:0.5"
+      useCustomGlobalHooks: true
+
+  - group: "Serverless"
+    key: serverless-integration-tests
+    depends_on:
+      - integration-serverless
+    steps:
+      - label: "Windows:2022:amd64:sudo"
+        depends_on:
+          - packaging-windows
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'packaging-windows'
+          .buildkite/scripts/integration-tests.ps1 fleet true TestLogIngestionFleetManaged serverless #right now, run a single test in serverless mode as a sort of smoke test, instead of re-running the entire suite
+        artifact_paths:
+          - build/**
+          - build/diagnostics/**
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
+          image: "family/platform-ingest-elastic-agent-windows-2022"
+        retry:
+          automatic:
+            limit: 1
+      - label: "Windows:2025:amd64:sudo"
+        depends_on:
+          - packaging-windows
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'packaging-windows'
+          .buildkite/scripts/integration-tests.ps1 fleet true TestLogIngestionFleetManaged serverless #right now, run a single test in serverless mode as a sort of smoke test, instead of re-running the entire suite
```

Comment:
```suggestion
          # right now, run a single test in serverless mode as a sort of smoke test, instead of re-running the entire suite
          .buildkite/scripts/integration-tests.ps1 fleet true TestLogIngestionFleetManaged serverless
```

Reply:
b1492eaca25ae811afb1db5e37249e3e5141c4c3 here you go 🙂